core/src/main/scala/org/http4s/ContextRequest.scala

```diff
@@ -17,4 +17,26 @@ object ContextRequest {
   def apply[F[_]: Functor, T](
       getContext: Request[F] => F[T]): Kleisli[F, Request[F], ContextRequest[F, T]] =
     Kleisli(request => getContext(request).map(ctx => ContextRequest(ctx, request)))
+
+  implicit def contextRequestInstances[F[_]]: NonEmptyTraverse[ContextRequest[F, *]] =
+    new NonEmptyTraverse[ContextRequest[F, *]] {
+      // Members declared in cats.Foldable
```

Comment:
These comments seem like something tooling can tell us. I don't think they add much.

Reply:
They were auto-generated by metals and I was too lazy to remove them. Will do.